src/cita/sourceItemWrapper.ts

```diff
@@ -223,23 +255,31 @@ class SourceItemWrapper extends ItemWrapper {
 				}
 			});
 		} else if (this._storage === "note") {
-			const note = Wikicite.getCitationsNote(this.item);
-			if (note) {
+			const notes = Wikicite.getCitationsNotes(this.item);
+			if (notes.length > 0) {
 				let parser;
-				try {
-					parser = new DOMParser();
-				} catch {
-					// Workaround fix Pubpeer compatibility issue #41
-					// @ts-ignore Components.classes[] isn't support by the types
-					parser = Components.classes[
-						"@mozilla.org/xmlextras/domparser;1"
-						// @ts-ignore the types don't include nsIDOMParser
-					].createInstance(Components.interfaces.nsIDOMParser);
-				}
-				const doc = parser.parseFromString(note.getNote(), "text/html");
-				const rawCitations = JSON.parse(
-					doc.getElementsByTagName("pre")[0].textContent,
-				);
+				const citationsString = notes
+					.map((note) => {
+						try {
+							parser = new DOMParser();
+						} catch {
+							// Workaround fix Pubpeer compatibility issue #41
+							// @ts-ignore Components.classes[] isn't support by the types
+							parser = Components.classes[
+								"@mozilla.org/xmlextras/domparser;1"
+								// @ts-ignore the types don't include nsIDOMParser
+							].createInstance(
+								Components.interfaces.nsIDOMParser,
+							);
+						}
+						const doc = parser.parseFromString(
+							note.getNote(),
+							"text/html",
+						);
+						return doc.getElementsByTagName("pre")[0].textContent;
+					})
+					.join("");
+				const rawCitations = JSON.parse(citationsString);
```

Comment:
`loadCitations` seems to be called whenever a (partial) citation note is saved, which means that until all of them have been saved, `citationsString` will be incomplete and `JSON.parse` will throw an error. I've wrapped it in a try/catch block locally, but it's not the best strategy. We should either find a way to "stall" item pane updates until all notes have been saved or maybe have a state variable for the SourceItemWrapper like `isSaving` or something that would be checked when loading citations.